aica_docs/set_up_reinforcement_learning.md

```diff
@@ -0,0 +1,563 @@
+   A Robotic arm learning to reach a specified end-effector position and orientation.
+
+2. [Lift Environment](../source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/manipulation/lift/lift_env_cfg.py)  
+   A Robotic arm learning to lift an object within the workspace.
+
+3. [Stack Environment](../source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/manipulation/stack/stack_env_cfg.py)  
+   A Robotic arm learning to stack multiple objects.
+
+For more details on complex reward terms, such as triggering penalties when an object falls out of the workspace, and on
+curriculum learning refer to the official
+[Isaac Lab Manager-Based RL Environment](https://isaac-sim.github.io/IsaacLab/main/source/tutorials/03_envs/create_manager_rl_env.html)
+documentation.
+
+## Reinforcement Learning Workflows
+
+Several open-source RL frameworks facilitate policy learning and optimization. Isaac Lab offers wrappers for various RL
+libraries, translating environment data into the appropriate formats for each library's functions. The three supported
+libraries are:
+
+1. `RSL-RL`: Developed by the Robotic Systems Lab at ETH Zürich, RSL-RL is a fast and straightforward implementation of
+   RL algorithms designed to run entirely on GPUs. It currently supports Proximal Policy Optimization (PPO), with plans
+   to incorporate additional algorithms. More details can be found on
+   [RSL-RL Github](https://github.com/leggedrobotics/rsl_rl).
+
+2. `SKRL`: An open-source modular RL library built on top of PyTorch and JAX. SKRL emphasizes modularity, readability,
+   and simplicity, supporting various environments, including NVIDIA Isaac Gym and Omniverse Isaac Gym. It enables
+   simultaneous training of multiple agents with customizable scopes. More details can be found on
+   [SKRL Github](https://github.com/Toni-SM/skrl).
+
+3. `RL Games`: An open-source high-performance RL framework designed for training policies in simulated environments.
+   More details can be found on [RL Games Github](https://github.com/Toni-SM/skrl).
+
+For detailed instructions on running the various RL libraries, refer to
+[Reinforcement Learning Wrappers](https://isaac-sim.github.io/IsaacLab/main/source/overview/reinforcement-learning/rl_existing_scripts.html).
+
+To prepare a RL environment for use with one of the mentioned wrappers, additional configuration files need to be
+specified. For an example, check out the
+[reach directory](../source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/manipulation/reach) in
+the Isaac Lab repository.
+
+### Step 1: Create a New Folder
+
+Begin by creating a new folder. Its location depends on the simulation environment and the specific task being
+performed. For example, for a Manager-Based RL robotic arm manipulation reach task, an appropriate location for that
```

Comment:
```suggestion
performed. For example, for a Manager-Based RL robotic arm manipulation reach task, an appropriate location for that
```

Here, for example, I'm ok(ish) with capitalizing `Manager-Based RL` only because NVidia is consistently going by this convention.